### PR TITLE
Marko pull request

### PR DESCRIPTION
**Pull request by popovicmarko**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *MarkoP* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Marko Popovic</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="_blank">@marko_popov1c</a>
```
Comment (TenTraicion) [inline index.html:123]:
```suggestion
            <a href="https://www.twitter.com/marko_popov1c" target="_blank">@marko_popov1c</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@popovicmarko, Thanks for your contribution. Kindly make the required change and commit.

**Comment by CandidDeer** [comment]
@popovicmarko Please note that open PRs with change requests will be closed 7 days after opening if the change requests have not been resolved by then. Thx.

**Review by popovicmarko — COMMENTED** [review]
I fixed what you asked for

**Comment by popovicmarko** [comment]
I've commited recommended changes

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@popovicmarko, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**